bioimageio/core/build_spec/build_model.py

```diff
@@ -595,7 +602,7 @@ def build_model(
     authors: List[Dict[str, str]],
     tags: List[Union[str, Path]],
     documentation: Union[str, Path],
-    cite: Dict[str, str],
+    cite: List[Dict[str, str]],
     output_path: Union[str, Path],
     # model specific optional
     architecture: Optional[str] = None,
```

Comment:
Would it make sense to deprecate the architecture and place it under a more generic weight kwargs field?